src/Calculators/Dominion/CasualtiesCalculator.php

```diff
@@ -334,6 +334,8 @@ protected function isImmortalVersusRacePerk(Dominion $dominion, string $opposing
         $raceNameFormatted = strtolower($opposingForceRaceName);
         $raceNameFormatted = str_replace(' ', '_', $raceNameFormatted);
```

Comment:
You can use `snake_case($opposingForceRaceName)` here :) Automatically lowercases and replaced spaces with underscores.